exercises/custom-set/custom_set_test.rb

```diff
@@ -3,114 +3,303 @@
 require 'minitest/autorun'
 require_relative 'custom_set'
 
+# rubocop:disable Metrics/LineLength
```

Comment:
This file is generated and the lines are only long because the test names are verbose so I have no problem with disabling the LineLength metric here.